modal_proto/api.proto

```diff
@@ -2242,10 +2242,13 @@ message SandboxGetLogsRequest {
 
 message SandboxGetTaskIdRequest {
   string sandbox_id = 1;
+  float timeout = 2;
+  bool wait_until_ready = 3; // If true, waits until the container's postStart hook has been run before returning. Useful for detecting init failures.
```

Comment:
Can this just be represented with `timeout = 0`?

Reply:
I don't think so. `wait_until_ready` does something different if it is true: it will wait until the task fully starts. If `wait_until_ready` is false, it will only wait until the task is scheduled, but it will not wait for the task to fully initialize.

Also, timeout = 0 represents "use the default timeout" for backwards-compatibility with old clients.